actions/transfer-case.vbs

```diff
@@ -135,24 +136,36 @@ DO
     IF active_worker_found = FALSE THEN msgbox err_msg
 LOOP UNTIL active_worker_found = TRUE
 
-transfer_out_of_county = FALSE                                          'setting varible to false
+transfer_out_of_county = FALSE                                          'setting variable to false
 IF left(transfer_to_worker, 4) <> "X127" THEN
     transfer_out_of_county = TRUE 'setting the out of county BOOLEAN
 Else
     '----------------------------------------------------------------------------------------------------In-county transfer
+    'Staff need to confirm that they've transferred the case in ECF Next 1st
+    Do 
+        Do         
+            transfer_confirmation = MsgBox("Has this case been transferred in ECF Next?", vbQuestion + vbYesNo, "Tranfer Case Reminder")
+            If transfer_confirmation = vbNo then msgbox "This case needs to be transferred in ECF Next first." & vbcr & vbcr & "Transfer the case in ECF Next now."
+            If transfer_confirmation = vbYes then exit do 
+        Loop 
+        CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
+    Loop until are_we_passworded_out = false					'loops until user passwords back in
+    
     CALL navigate_to_MAXIS_screen ("SPEC", "XFER")         'go to SPEC/XFER IN COUNTY
     Call write_value_and_transmit("X", 7, 16)              'transfer within county option
     PF9                                                    'putting the transfer in edit mode
     EMreadscreen second_servicing_worker, 7, 18, 74        'checking to see if the transfer_to_worker is the same as the second_servicing_worker (because then it won't transfer)
     IF second_servicing_worker <> "_______" THEN CALL clear_line_of_text(18, 74)
     Call write_value_and_transmit(transfer_to_worker, 18, 61)           'entering the worker information & saving - this should then take us to the transfer menu
     EMReadScreen servicing_worker, 7, 24, 30  'if it is not the transfer_to_worker - the transfer failed.
+    EMReadScreen transfer_message, 79, 24, 2
+    transfer_message = trim(transfer_message)
 
     STATS_manualtime = 120                      'manual run time in seconds
-    'Script low down for innercounty transfer
+    'Script low down for inner county transfer
     script_run_lowdown = script_run_lowdown & "transfer_out_of_county: " & transfer_out_of_county & vbCr & "worker_number: " & worker_number & vbCr & "transfer_to_worker: " & transfer_to_worker & vbCr & " Error Message at transfer: " & vbCr & error_message
     If servicing_worker <> transfer_to_worker THEN script_end_procedure_with_error_report("Transfer of this case to " & transfer_to_worker & " has failed.")
-    script_end_procedure_with_error_report("Success! Case transfer has been completed to: " & transfer_to_worker & ".")
+    script_end_procedure_with_error_report("Success! " & transfer_message)
```

Comment:
I think this might be an unclear message on it's own for the users. Helpful for us in reviewing any error reports, but it probably needs some context in the end message.

Reply:
The message does have some decent information. Here's an example from a training case: CASE XFER'D FROM X127GM1 TO X127L1S ON 01/23/24 AT 13:39:04

Reply:
I'm thinking something more like this:
![image](https://github.com/Hennepin-County/MAXIS-scripts/assets/13418322/13a11132-c34e-4844-b361-fd7c6b27d3eb)

This would also be great for if it failed as well. 

I just know that I have struggled getting users to understand when information is coming from MAXIS. I have a commit with some updates.